src/test/java/uk/ac/ebi/eva/commons/mongodb/repository/VariantRepositoryTest.java

```diff
@@ -275,6 +276,7 @@ public void testRegionIsFoundWithPolyphenGreaterThan() {
         testFiltersHelperRegion(regions, filters, exclude, 4);
     }
 
+    @Ignore
```

Comment:
were this ignores temporal? or there was no way to make them work? maybe some comment explaining why is it ignored would be helpfule. (there are other ignored tests in this file as well)